src/Test/Utilities/Portable/project.json

```diff
@@ -9,16 +9,14 @@
     "System.Diagnostics.Tools": "4.0.1",
     "System.IO.FileSystem": "4.0.1",
     "System.Reflection.TypeExtensions": "4.1.0",
+    "System.Text.RegularExpressions": "4.1.0",
```

Comment:
@ericstj: [before the change](https://gist.github.com/jasonmalinowski/2951cf451dcf51f901ed480303f7f921#file-before-project-lock-json-L573-L583) it was pulling in because of a dependency from another package. [After the change](https://gist.github.com/jasonmalinowski/2951cf451dcf51f901ed480303f7f921#file-before-project-lock-json-L573-L583) the package is still being referenced but instead I'm being handed _._ as the compile asset. Is there some difference in NuGet behavior where it'll only expose compile assets in some cases if you've explicitly referenced the package?